tests/utils/test_credentials.py

```diff
@@ -99,3 +102,28 @@ def test_read_mlflow_creds_env_takes_precedence_over_file(tmp_path, monkeypatch)
         creds = read_mlflow_creds()
         assert creds.username == "username_env"
         assert creds.password == "password_env"
+
+
+def test_mlflow_login(tmp_path, monkeypatch):
+    with patch(
+        "builtins.input", side_effect=["https://community.cloud.databricks.com/", "weirdmouse"]
+    ), patch("getpass.getpass", side_effect=["dummypassword"]):
+        monkeypatch.setenv("DATABRICKS_CONFIG_FILE", f"{tmp_path}/.databrickscfg")
+
+        class FakeWorkspaceClient:
+            class FakeClusters:
+                def list(self):
+                    return ["cluster1"]
+
+            def __init__(self):
+                self.clusters = FakeWorkspaceClient.FakeClusters()
+
+        with patch(
+            "databricks.sdk.WorkspaceClient",
+            side_effect=[
+                MlflowException("Error"),
+                FakeWorkspaceClient(),
+            ],
+        ):
+            login("databricks")
```

Comment:
can we read `f"{tmp_path}/.databrickscfg"` and ensure the host and token are included?

Reply:
good call